README.md

```diff
@@ -61,6 +61,9 @@ CREATE TABLE error_counts AS
 
 # Latest news
 
+* [KSQL Feb 2018 release available](https://www.confluent.io/blog/ksql-february-release-streaming-sql-apache-kafka/)
```

Comment:
I wonder if this can be a later commit. the blog isn't published yet, so this will be broken.